app/views/schedulers/index.html.erb

```diff
@@ -10,6 +10,7 @@
     <th>Next</th>
     <th>Interval</th>
     <th>Visual Run</th>
+    <th>Operations</th>
```

Comment:
Adding a `<th>` that corresponds to the operations column as currently there is no column header, which makes the table appear unsightly.